great_expectations/types/configurations.py

```diff
@@ -30,17 +19,3 @@ def module_name(self):
 class ClassConfigSchema(Schema):
     class_name = fields.Str()
     module_name = fields.Str(allow_none=True)
-
-#
-#     #
-#     # _allowed_keys = {
-#     #     "module_name",
-#     #     "class_name"
-#     # }
-#     # _required_keys = {
-#     #     "class_name"
-#     # }
-#     # _key_types = {
-#     #     "module_name": string_types,
-#     #     "class_name": string_types
-#     # }
```

Comment:
It's a red party!
![red_party](https://user-images.githubusercontent.com/928247/74055613-88151e80-499d-11ea-8c29-3359b5c7c3b3.gif)